app/ai/voice/agents/breeze_buddy/services/call_limiter.py

```
@@ -0,0 +1,119 @@
+                    {
+                        "name": "Window",
+                        "value": f"{window_seconds}s",
+                    },
+                    {"name": "Lead ID", "value": lead_id},
+                    {"name": "Reseller", "value": reseller_id},
+                    {"name": "Action", "value": action},
+                ],
+            )
+            return not block_enabled
+        return True
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)